### ISSUE TITLE
Circle Around Crosshair?

### ISSUE THREAD

**Issue by Excalviner** [omp/tf2hud#103]
I really would like a circle around my crosshair, but I don't know how to get it. I've read your website, looked through the issues, and searched the internet, but can't find anything. Could you help? :D

**Comment by fbis251** [comment]
From the website:
Crosshairs: Several crosshairs are defined at the very bottom of scripts/hudlayout.res. They can be enabled by changing the enabled and visible flags to 1. These crosshairs are centred for 1920x1080 and will most likely require adjustments at other resolutions. Play with the xpos, ypos, wide and tall values to adjust the position. To add an outline to any crosshair, add Outline to the font before the number; for example, Crosshairs20 would become CrosshairsOutline20.

Basically open hudlayout.res, find "OmpCrosshairCircle" and change visible and enabled to 1
